### PR TITLE
Proposal for Service Level Indicators (SLI) spec

### PR DESCRIPTION
**Pull request by johannes-b**

### CODE REVIEW

File: sre.md [hunk 1]
```diff
@@ -241,6 +242,52 @@ total_score:
...
+      },
+      "type": "object"
+    }
+  },
+  "additionalProperties": false,
+  "type": "object"
+}
+```
+
+### Example of a Service Level Indicators (SLO) configuration (in yaml)
```
Comment (christian-kreuzberger-dtx) [inline sre.md]: typo: (SLO) should be (SLI)

Comment (johannes-b) [reply sre.md]: thx!

File: sre.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
 # Specifications for Site Reliability Engineering with Keptn
 
 To support site reliability engineering with Keptn and to enable the self-healing use case, Keptn relies on two configurations called as:
-* [Service Level Objectives (SLO)](#service-level-objectives-(slo))
+* [Service Level Objectives (SLO)](#service-level-objectives-slo-configuration))
```
Comment (christian-kreuzberger-dtx) [inline sre.md]: there seems to be an extra closing `)` at the end - not caused by you PR, but can you still fix it please?

Comment (johannes-b) [reply sre.md]: It is gone.

**Pull request merged**